src/components/payment/payment-mixin.ts

```diff
@@ -174,10 +173,11 @@ export default class PaymentMixin extends Vue {
    * @param redirectUrl
    */
   redirectToPaymentPortal (paymentId, paymentToken, redirectUrl) {
-    const paymentPortalUrl = `${this.$PAYMENT_PORTAL_URL}/${paymentToken}/${redirectUrl}`
```

Comment:
I moved `this.$PAYMENT_PORTAL_URL` out of the Vue context and into session storage.